python/tests/job_runner/test_jobmanager.py

```diff
@@ -171,11 +171,11 @@ def test_run_multiple_ok(self):
         for status in statuses:
             self.assertEqual(status.exit_code, 0)
 
-        for index, dir_number in enumerate(dir_list):
-            self.assertTrue(os.path.isdir(dir_list[index]))
-            self.assertTrue(os.path.isfile("mkdir_out.%d" % index))
-            self.assertTrue(os.path.isfile("mkdir_err.%d" % index))
-            self.assertEqual(0, os.path.getsize("mkdir_err.%d" % index))
+        for dir_number in dir_list:
+            self.assertTrue(os.path.isdir(dir_number))
+            self.assertTrue(os.path.isfile("mkdir_out.%s" % dir_number))
+            self.assertTrue(os.path.isfile("mkdir_err.%s" % dir_number))
+            self.assertEqual(0, os.path.getsize("mkdir_err.%s" % dir_number))
```

Comment:
Would also prefer to use the `format` function here if we are touching the code anyways!